src/main/resources/entity-types/orders/simple_purchase_order_line.json5

```diff
@@ -0,0 +1,1625 @@
+              filterValueGetter: "( SELECT array_agg(lower(elems.value->>'contributorNameTypeId')) FROM jsonb_array_elements(:sourceAlias.jsonb->'contributors') AS elems)",
+              valueFunction: 'lower(:value)',
+            },
+          ],
+        },
+      },
+      isIdColumn: false,
+      queryable: false,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'contributors'",
+    },
+    {
+      name: 'cost_list_unit_price',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'numberType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'listUnitPrice'",
+    },
+    {
+      name: 'cost_list_unit_price_electronic',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'numberType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'listUnitPriceElectronic'",
+    },
+    {
+      name: 'cost_currency',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'currency'",
+    },
+    {
+      name: 'cost_additional_cost',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'numberType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'additionalCost'",
+    },
+    {
+      name: 'cost_discount',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'numberType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'discount'",
+    },
+    {
+      name: 'cost_discount_type',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'discountType'",
+      values: [
+        {
+          value: 'amount',
+          label: 'Amount',
+        },
+        {
+          value: 'percentage',
+          label: 'Percentage',
+        },
+      ],
+    },
+    {
+      name: 'cost_exchange_rate',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'numberType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'exchangeRate'",
+    },
+    {
+      name: 'cost_quantity_physical',
+      sourceAlias: 'pol',
+      dataType: {
+        dataType: 'integerType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'cost'->>'quantityPhysical'",
```

Comment:
We should cast this to an integer and add a valueFunction that casts the value to an integer